### PR TITLE
Adapted to new ipfs-repo API

### PR DESCRIPTION
**Pull request by pgte**
Still needs new relesease of ipfs-repo and ipfs-block-service.

### CODE REVIEW

**Comment by pgte** [comment]
Depends on https://github.com/ipfs/js-ipfs-repo/pull/140

File: src/index.js [hunk 1]
```diff
@@ -356,9 +356,24 @@ class IPLDResolver {
...
+      root: {
+        db: Memdown
+      },
+      blocks: {
+        db: Memdown
+      },
+      datastore: {
+        db: Memdown
+      }
+    },
```
Comment (diasdavid) [inline src/index.js]: @pgte is this correct?

Comment (pgte) [inline src/index.js]: @diasdavid looks like it. I think that, for the tests, it doesn't make a difference whether we use an extension or sharing. Wouldn't you agree?

Comment (pgte) [inline src/index.js]: and that, generally, without persistence, these options would be enough..

**Pull request merged**